doc/articles/debugging-wasm.md

```diff
@@ -36,10 +36,10 @@ Debugging WebAssembly via Google Chrome is experimentally supported by the Uno P
 > You may simply install _Chrome Beta_ or _Chrome Canary_ and use them instead.
 > * Sometimes, you may have a problem removing a breakpoint from code (it's crashing the debugger).
 > You can remove them in the _Breakpoints list_ instead.
-> * Once _IIS Express_ is launched, no need to press `Ctrl+F5` again: you simply need to rebuild your
-> _Wasm_ head and refresh it in the browser.
-> * **To refresh an app**, you should use the debugger tab and press the _refresh button_ in the content.
+> * Once _IIS Express_ is launched, no need to press <kbd>Ctrl</kbd><kbd>F5</kbd> again: you simply need to rebuild your
+> _WASM_ head and refresh it in the browser.
```

Comment:
```suggestion
> _Wasm_ head and refresh it in the browser.
```

Reply:
I think when used in a documentation context (e.g. not in code), it's spelled WASM, as acronyms, no?

Reply:
Like [LINQ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/). It's spelled _LINQ_, but coded as `Linq`.